### DISCUSSION TITLE
Helper script I used for reordering ALCOM translation keys

### DISCUSSION THREAD

**Discussion by Sayamame-beans** [vrc-get/vrc-get#1511] [Show and tell]
This is very late, but I did not post the helper script used for reordering keys, so here is a link to it. (I just published it.)
I don't know whether there will be a next run... anyway, I publish this.
https://gist.github.com/Sayamame-beans/849319b41497fa87a09e14661331119f

**Comment by Spokeek** [comment]
Thank you for the script, i'll try it see how it applies to the french file.
